inference.py

```diff
@@ -108,7 +109,44 @@ def __init__(self, ar_ckpt, nar_ckpt, device: str = None) -> None:
         self.vocos = Vocos.from_pretrained("charactr/vocos-encodec-24khz").to(self.device).eval()
         nuke_weight_norm(self.codec)
         nuke_weight_norm(self.vocos)
-        
+
+    @classmethod
+    def _from_pretrained(
+        cls: Type["Mars5TTS"],
+        *,
```

Comment:
docs? What does * mean? What can users specify there?

Reply:
that's snippet from HF itself